src/electron/views/automated-checks/automated-checks-view.scss

```diff
@@ -19,6 +19,12 @@
     -webkit-app-region: no-drag;
     background-color: $neutral-2;
 
+    .automated-checks-panel-container {
+        display: flex;
+        flex-direction: column;
+        overflow: hidden;
```

Comment:
Why is the overflow necessary?

Reply:
If you're seeing a visual issue, it'd be helpful if you could provide a gif. Just trying to understand here.

Reply:
![ai-unified-automated-checks-container-overflow](https://user-images.githubusercontent.com/48259897/94742425-87653a00-032a-11eb-87f8-0b569b790f9b.gif)
In the gif attached above, when `overflow: hidden` is not applied, the app ends up having a scrollbar for the whole window instead of separate vertical scrollbars for the instances cards panel and screenshot view.